### PR TITLE
GCPIngressParams CRD

### PR DESCRIPTION
**Pull request by swetharepakula**

### CODE REVIEW

File: pkg/apis/ingparams/v1beta1/types.go [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+	Spec   GCPIngressParamsSpec   `json:"spec,omitempty"`
+	Status GCPIngressParamsStatus `json:"status,omitempty"`
+}
+
+// GCPIngressParamsSpec is the spec for a GCPIngressParams resource
+// +k8s:openapi-gen=true
+type GCPIngressParamsSpec struct {
+	// External specifies whether external or internal load balancing is desired
+	// +required
+	External bool `json:"external"`
```
Comment (freehan) [inline pkg/apis/ingparams/v1beta1/types.go:41]: you probably want to add a comment and say the default is internal

Comment (swetharepakula) [reply pkg/apis/ingparams/v1beta1/types.go:41]: That's a good point! I will switch to match current behavior.

**Review by freehan — APPROVED** [review]
/lgtm
/approve

**Pull request merged**